lib/modules/manager/mix/extract.ts

```diff
@@ -8,15 +8,17 @@ const depSectionRegExp = regEx(/defp\s+deps.*do/g);
 const depMatchRegExp = regEx(
   /{:(?<depName>\w+),\s*(?<datasource>[^:"]+)?:?\s*"(?<currentValue>[^"]+)",?\s*(?:organization: "(?<organization>.*)")?.*}/gm
 );
+const commentMatchRegExp = regEx(/^\s*#/);
 
 export async function extractPackageFile(
   content: string,
   fileName: string
 ): Promise<PackageFile | null> {
   logger.trace('mix.extractPackageFile()');
   const deps: PackageDependency[] = [];
-  const contentArr = content.split(newlineRegex);
-
+  const contentArr = content
+    .split(newlineRegex)
+    .filter((line) => !commentMatchRegExp.test(line));
   for (let lineNumber = 0; lineNumber < contentArr.length; lineNumber += 1) {
     if (contentArr[lineNumber].match(depSectionRegExp)) {
       logger.trace(`Matched dep section on line ${lineNumber}`);
```

Comment:
this should be removed, as linenumer is now wrong when comment lines are skipped 😕